lib/main_containers.pm

```diff
@@ -277,6 +277,9 @@ sub load_container_tests {
                 load_image_tests_in_k8s($run_args) if (/k8s/i);
                 load_image_tests_in_openshift if (/openshift/i);
             }
+        } elsif (get_var('PODMAN_BATS')) {
```

Comment:
Does it make sense to do this here, inside the loop for every `CONTAINER_RUNTIME`? Treating this more like an `EXTRATEST` or moving it before the loop probably matches better

Reply:
Fixed.